code/paul/HTML_and_CSS/BLOG_LAB/blog.py

```diff
@@ -0,0 +1,52 @@
+#Step one- Hello World (Flask Notes for Class; Flask Docs)-----> Lines 02-06
+
+# @app.route('/')
+# def index():
+#         return 'Hello World!'
+# app.run(debug=True)
+
+
+from flask import Flask, render_template
+app= Flask(__name__)
+
+@app.route('/')
+def index():
+        return 'Hello World!'
+
+@app.route('/bloglab')
```

Comment:
How does the user know to get here? When the user goes to your website they get a single page that says `Hello World`.